index.html

```diff
@@ -1692,6 +1692,29 @@ <h2>Example of filter output</h2>
     </div>
     <!-- ends Side by Side Videos/Temporal Difference Filter -->
 
+    <!-- xstack -->
+    <label class="recipe" for="xstack">Use xstack to arrange output layout of multiple video sources</label>
+    <input type="checkbox" id="xstack">
+    <div class="hiding">
+      <h3>This filter allows for the vertical and horizontal stacking of multiple video sources into one output.</h3>
```

Comment:
I am not English speaker, but would have said: `This filter allows vertical and horizontal stacking`.

Reply:
I think that 'allows for' actually conveys a particular nuance over 'allows' - BUT good point that it is a little wordy. `Enables` is more concise!

Reply:
Thank you for the explanation.